### ISSUE TITLE
[New Primitive] `Tag` / `Chip`

### ISSUE THREAD

**Issue by kaushalyap** [radix-ui/primitives#987]
### Overview

Tag component like in StackOverflow which is closable.

### Examples in other libraries

https://mui.com/components/chips/

### Who does this impact? Who is this for?

Beginners or anyone who want to save time.

### Additional context

N/A

**Comment by jjenzz** [comment]
I'm not sure how this sits in Radix because Radix provides functional/accessible **patterns** without styles but this appears to be purely a styling thing. Each of those examples there are just various component types **styled to look like a chip**.

- [Basic Chip](https://mui.com/components/chips/#basic-chip) - A `div` styled as a chip
- [Clickable Chip](https://mui.com/components/chips/#clickable) - A `button` styled as a chip that you attach `onClick` to
- [Deletable Chip](https://mui.com/components/chips/#deletable) - A `button` styled as a chip that you attach `onClick` to
- [Link Chip](https://mui.com/components/chips/#clickable-link) - An `a` styled as a chip
- [Avatar Chip](https://mui.com/components/chips/#avatar-chip) - Our `Avatar` styled as a chip

... etc. 

Would you mind elaborating on what you think Radix could provide here?

**Comment by benoitgrelard** [comment]
The only "functionality" I can see in the MUI one is a little bit of keyboard support (the delete functionality via backspace for example).

**Comment by jjenzz** [comment]
Yeah I could maybe see a case for a `TagGroup` type of component that encapsulates this example https://mui.com/components/chips/#chip-array

My only thinking though is that tags are often a small part of a bigger pattern like the following, so I wonder if a `TagGroup` might be too low level?

![image](https://user-images.githubusercontent.com/175330/142635411-75f4895c-8c4a-462f-9d17-91c12e4ac695.png)

**Comment by benoitgrelard** [comment]
Yeah this is usually more like a combobox with multiple selection, something like that.

**Comment by jjenzz** [comment]
I randomly just came across this in the a11y slack too. It seems these type of tags should be labelled by the thing they're tagging so there are likely some differing accessibility requirements depending on the use-case.

![image](https://user-images.githubusercontent.com/175330/142639264-f7476706-4343-4673-ae94-17e4dfbed8ce.png)

**Comment by kaushalyap** [comment]
> ![image](https://user-images.githubusercontent.com/175330/142635411-75f4895c-8c4a-462f-9d17-91c12e4ac695.png)

Needs something like that like in SO.

**Comment by gary-lo** [comment]
Any updates on this?

**Comment by SalahAdDin** [comment]
> Any updates on this?

Nothing yet.

**Comment by elrumordelaluz** [comment]
related: https://www.radix-vue.com/components/tags-input

**Comment by YusufcanY** [comment]
Any updates? Really need this primitive

**Comment by suitux** [comment]
Any news?

**Comment by aahnik** [comment]
Yepp, this will be a very helpful one!

**Comment by SalahAdDin** [comment]
> Yepp, this will be a very helpful one!
It is now not planned.
